bdb/attr.h

```diff
@@ -357,7 +357,7 @@ DEF_ATTR_2(
 DEF_ATTR(PAGEDEADLOCK_MAXPOLL, pagedeadlock_maxpoll, QUANTITY, 5,
          "If retrying on deadlock (see pagedeadlock_retries), poll up to this "
          "many ms on each retry.")
-DEF_ATTR(ENABLE_TEMPTABLE_CLEAN_EXIT, temp_table_clean_exit, BOOLEAN, 0,
+DEF_ATTR(ENABLE_TEMPTABLE_CLEAN_EXIT, temp_table_clean_exit, BOOLEAN, 1,
```

Comment:
This seems unnecessary ...

Reply:
It is only necessary to cleanup memory on exit. Not necessary otherwise.

Reply:
Why is it necessary to clean up temptables on clean exit?

Reply:
The goal is to have clean memory as reported by valgrind. I suppose we could only do so on DEBUG build, instead of Release build. Not sure it is worth spending time to differentiate tho.